lib/projects/localDev/AppDevModeInterface.ts

```diff
@@ -139,6 +165,10 @@ class AppDevModeInterface {
       return;
     }
 
+    if (this.appNode.config.auth.type === APP_AUTH_TYPES.STATIC) {
+      return installAppPrompt(await this.getAppInstallUrl(), false);
+    }
```

Comment:
This always prompts the user to install the app into portal when it is static auth